rmgpy/data/kinetics/database.py

```diff
@@ -458,7 +458,6 @@ def generate_reactions_from_families(self, reactants, products=None, only_famili
         # Delete reactions which we could not find a reverse reaction for
         for i in reversed(to_delete):
             del reaction_list[i]
-
```

Comment:
Could you remove this line deletion from this commit? Since this PR has no more changes to databse.py, it will create an unnecessary change to the history.